UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Idoit.API.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using Result = Idoit.API.Client.CMDB.ObjectTypes.Response.Result;
+using ObjectTypesRead = Idoit.API.Client.CMDB.ObjectTypes.ObjectTypes;
+using ObjectTypesCreate = Idoit.API.Client.CMDB.Object.Object;
+using Categories = Idoit.API.Client.CMDB.ObjectTypeCategories.ObjectTypeCategories;
+using ObjectTypeCategoriesResult = Idoit.API.Client.CMDB.ObjectTypeCategories.Response.Result;
+using Customs = Idoit.API.Client.CMDB.ObjectTypeCategories.Response.Customs;
+using System.IO;
+
+namespace UnitTestApi.CMDB.ObjectTypeCategories
+{
+    [TestClass]
+    public class ObjectTypeCategoriesTest
+    {
+        string URL;
+        string APIKEY;
+        string LANGUAGE;
+        public ObjectTypeCategoriesTest()
+        {
+            string path = Path.Combine(Environment.CurrentDirectory, @"Data\", "Api.env");
+            DotNetEnv.Env.Load(path); URL = DotNetEnv.Env.GetString("URL");
+            APIKEY = DotNetEnv.Env.GetString("APIKEY");
+            LANGUAGE = DotNetEnv.Env.GetString("LANGUAGE");
+        }
+        //Read
+        [TestMethod]
+        public void ReadTest()
+        {
+            string costumId = "";
+            string objectTypeId = "";
+            List<Result[]> lists = new List<Result[]>();
```

Comment:
"lists" is a very general name. Is it possible to give this a more speaking name?